pkg/controllers/disruption/singlenodeconsolidation.go

```diff
@@ -66,7 +66,7 @@ func (s *SingleNodeConsolidation) ComputeCommand(ctx context.Context, candidates
 	for i, candidate := range candidates {
 		if s.clock.Now().After(timeout) {
 			deprovisioningConsolidationTimeoutsCounter.WithLabelValues(singleMachineConsolidationLabelValue).Inc()
-			deprovisioningConsolidationTimeoutsCounter.WithLabelValues(s.ConsolidationType()).Inc()
+			disruptionConsolidationTimeoutTotalCounter.WithLabelValues(s.ConsolidationType()).Inc()
```

Comment:
Can we call the title of this pr to be a fix to add in this metric, rather than removing the duplicate? I think that'd more accurately reflect the impact.